LiskHQ/lisk-nano
# Write unit tests for electron code - Closes #900

### What was the problem?
We had no tests for the electron code

### How did I fix it?
Refactored and added unit ests

### How to test it?
- Run tests
- Start the electron app and check that it loads without problems
- Check the menu
- Check language can still be changed etc

### Review checklist
- [x] The PR solves https://github.com/LiskHQ/lisk-nano/issues/900
